### ISSUE TITLE
IE7 stomps PUT/DELETE/OPTIONS/HEAD via xhr -- Fix Included

### ISSUE THREAD

**Issue by nebulous** [jquery-form/form#58]
IE7 converts the form method attribute in the DOM to "get" if it is not set to GET or POST. Unfortunately, this means that $(form).attr('method') returns the incorrect, modified version. PUT/DELETE/etc should in-fact work just fine in IE7 when submitted via the XMLHttpRequest object. A fix is to ask for the 'method' attribute on the original DOM element, not the jQuery copy which contains the incorrect value courtesy of IE:

A fix is to add this to the beginning of $.fn.ajaxSubmit ~ line 46 or just before $.ajax(options) ~line 170

```
if ($form.attr('method') == 'get') { //Make certain IE hasn't modified the method attribute
    options.type = form.get(0).getAttribute('method'); 
}
```

That should un-stomp Internet Explorer's retardation and allow RESTy form submission.

**Comment by nebulous** [comment]
I've committed a more refined version of this fix and submitted pull request. Until that's accepted, feel free to get it here:  
https://github.com/nebulous/form/commit/f4e8ed21226fa695ae7633396b9698472ec9fe52

**Comment by malsup** [comment]
Fixed in v2.60 (I think)
